gdsfactory/config.py

```diff
@@ -223,6 +223,7 @@ class Settings(BaseSettings):
     on_type_missmatch: Literal["warn", "error", "ignore"] = Field(
         default="warn", description="When connecting ports with different types."
     )
+    default_show_suffix: Literal[".oas", ".gds"] = ".gds"
```

Comment:
This appears to be a global variable - can you give an example showing how to set it?

Reply:
You can do this

```python
import gdsfactory as gf

gf,CONF.default_show_suffix = '.oas'

```

or create a `gfconfig.yml` on your current working directory